cmd/bbolt/main.go

```diff
@@ -170,6 +170,7 @@ The commands are:
     pages       print list of pages with their types
     page-item   print the key and value of a page item.
     stats       iterate over all pages and generate usage stats
+    inspect     inspect the structure of the database
```

Comment:
Just curious: was there a reason not to add this to the [`Run` method](https://github.com/etcd-io/bbolt/blob/92c7414587095dfe0602f9de9c76ad8a9489518e/cmd/bbolt/main.go#L111) as well?  

Currently it seems the `inspect` command does not work, but the code is there thanks to this commit. trying to understand if that's intentional

Reply:
When the commands are not found in the [Run](https://github.com/etcd-io/bbolt/blob/92c7414587095dfe0602f9de9c76ad8a9489518e/cmd/bbolt/main.go#L111) method, bbolt will automatically try to run the cobra commands,

https://github.com/etcd-io/bbolt/blob/92c7414587095dfe0602f9de9c76ad8a9489518e/cmd/bbolt/main.go#L68-L69



> Currently it seems the `inspect` command does not work

Have you tried it out?  This isn't the first time I see this false statement. Please try it out, let me know whether you really have issue.

Reply:
Using `1.3.11` I get:

```
$ bbolt inspect /tmp/meta.db
unknown command
```

Reply:
Ok it seems the latest released version v1.3.11 does not have the changes you referenced (https://github.com/etcd-io/bbolt/blob/d128a10000a9d394686cf45be262a4fe966b03c4/cmd/bbolt/main.go#L69). It was confusing though because the [README](https://github.com/etcd-io/bbolt/blob/main/cmd/bbolt/README.md) references this command. I suspect that's why others are tripping up as well

Reply:
New features, including this one, only go into `main` branch.

So please try the main branch or [v1.4.0-beta.0](https://github.com/etcd-io/bbolt/releases/tag/v1.4.0-beta.0)

Reply:
Refer to https://github.com/etcd-io/bbolt/blob/main/CHANGELOG/CHANGELOG-1.4.md